### PR TITLE
feat(vendor-core): add graphql client packages

### PR DESCRIPTION
**Pull request by xprazak2**
Adds client packages for graphql which allow
us to fetch data using graphql.

ISSUES: refs #31745

### CODE REVIEW

**Comment by xprazak2** [comment]
Test failure unrelated - patternfly-core makes CustomContextSelector snapshots fail on new [prop for FocusTrap]( https://github.com/patternfly/patternfly-react/commit/1bc13185f8919a75f4fa1a0523e9f56a3d172dba#diff-264b31d642bbca579b76fb429cd47bf81e8a4f0488846fe43b45e771822220d4)

Pinning pf-core here has no effect as `npm install` inside foreman pulls vendor-core version specified in foreman's package.json which in turn pulls the pf-core version causing problems.

File: packages/vendor-core/package.json [hunk 1]
```diff
@@ -47,6 +48,8 @@
     "dsmorse-gridster": "^0.8.0",
     "file-saver": "^2.0.1",
     "formik": "^1.5.8",
+    "graphql": "^15.5.0",
+    "graphql-tag": "^2.11.0",
```
Comment (sharvit) [inline packages/vendor-core/package.json:52]: Shouldn't it suppose to be supplied by `@apollo/client`?

Comment (xprazak2) [reply packages/vendor-core/package.json]: Correct, removed.

**Comment by xprazak2** [comment]
Yes, all passing again (except for that one pesky test).

**Review by sharvit — APPROVED** [review]
Thanks @xprazak2 LGTM 👍

**Pull request merged**